docs/core/actions.rst

```diff
@@ -137,28 +137,51 @@ events in :ref:`Events <events>`.
+There are eight default actions:
+
++-----------------------------------+------------------------------------------------+
+| ``action_listen``                 | Stop predicting more actions and wait for user |
+|                                   | input.                                         |
++-----------------------------------+------------------------------------------------+
+| ``action_restart``                | Reset the whole conversation. Can be triggered |
+|                                   | during a conversation by entering ``/restart`` |
+|                                   | if the :ref:`mapping_policy` is included in    |
+|                                   | the policy configuration.                      |
++-----------------------------------+------------------------------------------------+
+| ``action_default_fallback``       | Undo the last user message (as if the user did |
+|                                   | not send it and the bot did not react) and     |
+|                                   | utter a message that the bot did not           |
+|                                   | understand. See :ref:`fallback-actions`.       |
++-----------------------------------+------------------------------------------------+
+| ``action_deactivate_form``        | Deactivate the active form and reset the       |
+|                                   | requested slot.                                |
+|                                   | See also :ref:`section_unhappy`.               |
++-----------------------------------+------------------------------------------------+
+| ``action_revert_fallback_events`` | Revert events that occurred during the         |
+|                                   | TwoStageFallbackPolicy.                        |
+|                                   | See :ref:`fallback-actions`.                   |
++-----------------------------------+------------------------------------------------+
+| ``action_default_ask_affirmation``| Ask the user to affirm their intent.           |
+|                                   | It is suggested to overwrite this default      |
+|                                   | action with a custom action to have more       |
+|                                   | meaningful prompts.                            |
++-----------------------------------+------------------------------------------------+
+| ``action_default_ask_rephrase``   | Ask the user to rephrase their intent.         |
++-----------------------------------+------------------------------------------------+
+| ``action_back``                   | Undo the last user message (as if the user did |
+|                                   | not send it and the bot did not react).        |
+|                                   | Can be triggered during a conversation by      |
+|                                   | entering ``/back`` if the MappingPolicy is     |
```

Comment:
also link here

Reply:
I think that would clutter it too much. It's literally the same name as above and used in a sentence. (I only included two links it for the Fallback since it is only a reference.)